scripts/release.sh

```diff
@@ -193,7 +213,7 @@ main() {
     log_warning "Skipping release build step. /release directory already exists."
   else
     log_callout "Building release..."
-    REPOSITORY=$(pwd) ./scripts/build-release.sh "${RELEASE_VERSION}"
+    REPOSITORY=$(pwd) CURRENT_BRANCH=${CURRENT_BRANCH} ./scripts/build-release.sh "${RELEASE_VERSION}"
```

Comment:
We just tagged locally release and we want to build it. I don't think passing here CURRENT_BRANCH=1 would do guarantee building the just tagged branch. Please double check.

Reply:
When CURRENT_BRANCH is 1, build_binary.sh will not clone the repo, and checkout the branch. Instead, it just builds the binaries from local current directory.

Reply:
`CURRENT_BRANCH` means current branch, so no `clone`, no `checkout`.